### ISSUE TITLE
--apache doesn't work on RHEL9, need to use `httpd` rather than `apachectl`

### ISSUE THREAD

**Issue by alexzorin** [certbot/certbot#9386]
Reported here: https://community.letsencrypt.org/t/redhat-9-apache-misconfigure/183455?u=_az.

**Comment by wgreenberg** [comment]
relevant changelog notes here (ctrl+f "apache") https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.0_release_notes/new-features#enhancement_dynamic-programming-languages-web-and-database-servers

seems like the apache `FedoraConfigurator` already handles this, should we just add a similar one for RHEL?

**Comment by alexzorin** [comment]
Yeah sounds good to me. Something like this for RHEL >= 9?

https://github.com/certbot/certbot/blob/a81d58fa6e3ce451682529947edfd807dcd56b7b/certbot-apache/certbot_apache/_internal/entrypoint.py#L51-L54

**Comment by bmw** [comment]
I think https://github.com/certbot/certbot/issues/8945 may be related.

**Comment by mobeigi** [comment]
Would be great if this was fixed! My only main issue on RHEL9.

So is the proposed solution to add a new `CentOSConfigurator` for RHEL >= 9 which uses the `httpd` equivalents instead of `apachectl`?

**Comment by alexzorin** [comment]
FWIW CentOS Stream 9 appears to behave the same way.

**Comment by invoicer-it** [comment]
I'm also eagerly waiting for this to get fixed on Centos Stream 9 (epel package at v1.30.0 as installed with `yum install certbot python3-certbot-apache`).

If anyone is in need of a hacky but gets-the-job-done patch you can apply these modifications (you'll still see errors, but it works).
File: `/usr/lib/python3.9/site-packages/certbot_apache/_internal/override_centos.py`
Diffs before after:
```
27,28c27,28
<         ctl="apachectl",
<         version_cmd=['apachectl', '-v'],
---
>         ctl="httpd",
>         version_cmd=['httpd', '-v'],
31c31
<         conftest_cmd=['apachectl', 'configtest'],
---
>         conftest_cmd=['httpd', '-t'],
75c75
<         self.options.restart_cmd_alt[0] = self.options.ctl
---
>         #self.options.restart_cmd_alt[0] = self.options.ctl
```
